akka-http/src/main/scala/akka/http/javadsl/server/directives/HeaderDirectives.scala

```diff
@@ -34,7 +35,11 @@ abstract class HeaderDirectives extends FutureDirectives {
    * @group header
    */
   def checkSameOrigin(allowed: HttpOriginRange, inner: jf.Supplier[Route]): Route = RouteAdapter {
```

Comment:
Ahh, I see :) Could you give a tip on how to do it in the best way or where to find an example?